<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewCustomer</class>
 <widget class="QWidget" name="AddNewCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddNewCustomer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2c3e50;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    color: #ecf0f1;
}

QGroupBox {
    border: 2px solid #3498db;
    border-radius: 10px;
    margin-top: 10px;
    color: white;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
    background-color: #2c3e50;
	
}

QLabel {
    color: white;
}

QPushButton#logoutButton {
    background-color: #e74c3c;
    color: white;
    font-size: 14px;
    padding: 8px 16px;
    border: none;
    border-radius: 8px;
}



QLineEdit , QSpinBox {
    background-color: #ecf0f1;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 5px;
	color:rgb(0, 0, 0)
}

QPushButton {
    background-color: #27ae60;
    color: white;
    padding: 8px 12px;
    font-weight: bold;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #2ecc71;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Add New Customer/Admin</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <widget class="QPushButton" name="Add_btn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QPushButton" name="logoutButton">
          <property name="text">
           <string>Back To Dashboard</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Customer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Admin</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Fname_LineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LName_LineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>National ID Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ID_LineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Username_LineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Password_LineEdit">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Day</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Month</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Year</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="Day_SpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>31</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Month_SpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Year_SpinBox">
              <property name="minimum">
               <number>1300</number>
              </property>
              <property name="maximum">
               <number>1404</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
